.dockerignore

```diff
@@ -0,0 +1,7 @@
+.github
+.git
+docker
+.dockerignore
+*.Rmd
+README.md
```

Comment:
I'd leave README.md, LICENSE -- very small files. Other than that, ready to merge I think.

Reply:
sure, I will no longer remove them